docs/components/input-checkbox.md

```diff
@@ -0,0 +1,127 @@
+# Checkbox
+
+**KCheckbox** - KCheckbox is a wrapper around a Kong styled checkbox input.
+
+<KCard>
+  <KCheckbox slot="body" v-model="defaultChecked"/>
+</KCard>
+
+```vue
+<template>
+  <KCheckbox
+    v-model="checked"
+    @change="handleToggle" />
+</template>
+<script>
+export default {
+  data() {
+    return {
+      checked: false
+    }
+  },
+  methods: {
+    handleToggle(isChecked) {
+      // do something, make api call?
+    }
+  }
+}
+</script>
+```
+
+## Props
+### v-model - required
+Use `v-model` to bind the `checked` state of the underlying `<input />`. The
+`v-model` binds to the `value` prop of the component and sets current checked
+state of the input. You can read more about passing values via `v-model`
+[here](https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components).
+
+```vue
+<KCheckbox v-model="checked" />
+```
+
+### label
+
+Will place label text to the right of the input. Can also be [slotted](#slots).
+
+- `label`
+
+```vue
+<KCheckbox
+  v-model="checked"
+  :label="checked ? 'on' : 'off'" />
```

Comment:
can you add a "multiple checkboxes" example? we've had bugs in the past where you check a label on a checkbox and it unchecks all the checkboxes...